Lavinia-api/Startup.cs

```diff
@@ -88,6 +97,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 options.DocumentTitle = "Lavinia API - Swagger";
             });
             app.UseStaticFiles();
+            app.UseIpRateLimiting();
```

Comment:
How sure are we about the ordering of this in [the pipeline](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/middleware/?view=aspnetcore-3.1)?

Reply:
The examples I have looked at only has the AddMvc call in addition to the rate limiting calls, and it appears before them.